test/check-docker

```diff
@@ -110,36 +110,37 @@ class TestDocker(MachineCase):
         b.wait_visible("#select-linked-containers:empty");
         b.click("#link-containers");
         b.wait_visible("#select-linked-containers:parent");
-        b.set_val("#select-linked-containers form:last-child select.selectpicker", "PROBE")
+        b.set_val("#select-linked-containers form:last-child select.selectpicker", "PROBE1")
         b.set_val("#select-linked-containers form:last-child  input[name='alias']", "alias1")
         b.click("#select-linked-containers form:last-child  button.fa-plus");
-        b.set_val("#select-linked-containers form:last-child  select.selectpicker", "PROBE")
+        b.set_val("#select-linked-containers form:last-child  select.selectpicker", "PROBE1")
         b.set_val("#select-linked-containers form:last-child  input[name='alias']", "alias2")
         b.click("#containers-run-image-run");
         b.wait_popdown("containers_run_image_dialog")
         b.wait_in_text("#containers-containers", "PROBE2")
+        b.wait_present("#containers-containers tbody tr:contains('PROBE2'):contains('Stopped')")
 
         # Check links
         info = json.loads(m.execute('docker inspect PROBE2'))
         self.assertEqual(set(info[0]["HostConfig"]["Links"]),
-                         set(["/PROBE:/PROBE2/alias1", "/PROBE:/PROBE2/alias2"]))
+                         set(["/PROBE1:/PROBE2/alias1", "/PROBE1:/PROBE2/alias2"]))
 
         # delete PROBE2
-        b.wait_in_text('#containers-containers tbody tr:first', "PROBE2")
-        b.wait_not_visible("#containers-containers tbody tr:first button.btn-delete")
+        b.wait_in_text('#containers-containers tbody', "PROBE2")
+        b.wait_not_visible("#containers-containers tbody tr:contains('PROBE2') button.btn-delete")
```

Comment:
Can't find an error here either, my mistake. I guess we have a race instead. Our preconditions for delete to become available are met before it is disabled again in the ui.